packages/azure-http-specs/specs/azure/resource-manager/operation-templates/available-operations.tsp

```diff
@@ -0,0 +1,39 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/spector";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+using Spector;
+
+namespace Azure.ResourceManager.OperationTemplates;
+
+@scenario("ListAvailableOperations")
+@scenarioDoc("""
+    Resource GET operation.
+    Expected path: /providers/Azure.ResourceManager.Models.Resources/operations
```

Comment:
Is this path correct?

Reply:
Fixed by the [commit#627a90e](https://github.com/Azure/typespec-azure/pull/1933/commits/627a90e082e6e2488ebd5459716755936fefbd92)